packages/contracts-periphery/contracts/universal/citizen-house/Badge.sol

```diff
@@ -0,0 +1,134 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { SafeTransferLib } from "@rari-capital/solmate/src/utils/SafeTransferLib.sol";
+import { ERC721 } from "@rari-capital/solmate/src/tokens/ERC721.sol";
+import { IERC165 } from "@openzeppelin/contracts/utils/introspection/IERC165.sol";
+import { Ownable } from "@openzeppelin/contracts/access/Ownable.sol";
+
+/**
+ * @title Badge contract
+ * @notice A minimalist soulbound ERC-721 Badge implementation
```

Comment:
Do you mind adding a note here in the doc about the permissions of this contract? Who is meant to be the owner vs who is meant to be the admin? Without this knowledge, its difficult to give sufficient review on the security of this contract

Reply:
Updated with small blurb - could probably use more!